updatehub/tests/successful_integration_test.rs

```diff
@@ -313,9 +321,8 @@ fn correct_config_update_no_polling_probe_api() {
     <timestamp> DEBG 70% of the file has been downloaded
     <timestamp> DEBG 90% of the file has been downloaded
     <timestamp> DEBG 100% of the file has been downloaded
-    <timestamp> INFO installing update: 4304291fa4d86ba6f924b3385ee1c0d5b4a0f8985abf3b84df73da3e5182ff0b
+    <timestamp> INFO installing update: 14aa3cb34ccd7047b81316ba42e5766b8eaeb2ddb32d537856e2819aacdf8a06
     <timestamp> INFO using installation set as target 1
-    <timestamp> DEBG running default check_requirements
```

Comment:
Is this intended? this does not seem related to this change.

Reply:
Yes, as intended. Because now the object has his own implementation of check requirements.

Reply:
Ahhh, makes sense.